mythril/ether/ethcontract.py

```diff
@@ -17,7 +17,8 @@ def __init__(self, code, creation_code="", name="Unknown", enable_online_lookup=
         code = re.sub(r'(_+.*_+)', 'aa' * 20, code)
 
         self.code = code
-        self.disassembly = Disassembly(self.code, enable_online_lookup=enable_online_lookup)
+        self.disassembly = Disassembly(code, enable_online_lookup=enable_online_lookup)
+        self.creation_disassemble = Disassembly(creation_code, enable_online_lookup=enable_online_lookup)
```

Comment:
disassembly

Reply:
shouldn't it be self.creation_disassembly?

Reply:
Oops I thought I changed it

Reply:
Wait! It’s changed right?